layouts/partials/posts/series.html

```diff
@@ -0,0 +1,23 @@
+{{ $currentPageUrl := .URL }}
+{{ if .Params.series }}
+<section class="see-also">
+  {{ range .Params.series }}
+    <h3>See also in {{ . }}</h3>
+    {{ $name := . | urlize }}
+    {{ $series := index $.Site.Taxonomies.series $name }}
+    {{ if gt (len $series.Pages) 0 }}
+      <nav class="list">
+        <ul>
+        {{ range first (add $.Site.Params.seriesRelatedCount 1 | default 6) $series.Pages }}
+          {{ if ne .URL $currentPageUrl }}
+            <li>
+              <a class="title" href="{{ .Params.ExternalLink | default .URL }}">{{ .Title }}</a>
```

Comment:
```suggestion
              <a href="{{ .Params.ExternalLink | default .URL }}">{{ .Title }}</a>
```
Remove this class too. The list looks nicier with the link look.